### PR TITLE
SQL_1311: Add custom connector documentation

### PR DESCRIPTION
**Pull request by nbagnard**
Microsoft requires us to provide a connector documentation following their guidelines defined here : https://learn.microsoft.com/en-us/power-query/providing-user-documentation

This is the .md file to share with them.

### CODE REVIEW

File: mongodb-atlas-sql-interface.md [hunk 1]
```diff
@@ -40,22 +40,25 @@ To connect using the Atlas SQL interface :
 
 3. Select **Database** from the categories on the left, select **MongoDB Atlas SQL**, and then select **Connect**.
 
-2. In the MongoDB Atlas SQL window that appears provide:
+4. If this is the first time you're connecting to MongoDB Atlas SQL connector, a third-party notice will be displayed. 
+   Select Don't warn me again with this connector if you don't want this message to be displayed again,
```
Comment (terakilobyte) [inline mongodb-atlas-sql-interface.md:44]:
```suggestion
   Select **"Don't warn me again with this connector"** if you don't want this message to be displayed again.
```

File: mongodb-atlas-sql-interface.md [hunk 2]
```diff
@@ -40,22 +40,25 @@ To connect using the Atlas SQL interface :
 
 3. Select **Database** from the categories on the left, select **MongoDB Atlas SQL**, and then select **Connect**.
 
-2. In the MongoDB Atlas SQL window that appears provide:
+4. If this is the first time you're connecting to MongoDB Atlas SQL connector, a third-party notice will be displayed. 
+   Select Don't warn me again with this connector if you don't want this message to be displayed again, 
+   and then select **Continue**.
```
Comment (terakilobyte) [inline mongodb-atlas-sql-interface.md:45]:
```suggestion

   Select **Continue**. 
```

File: mongodb-atlas-sql-interface.md [hunk 3]
```diff
@@ -40,22 +40,25 @@ To connect using the Atlas SQL interface :
 
 3. Select **Database** from the categories on the left, select **MongoDB Atlas SQL**, and then select **Connect**.
 
-2. In the MongoDB Atlas SQL window that appears provide:
+4. If this is the first time you're connecting to MongoDB Atlas SQL connector, a third-party notice will be displayed. 
+   Select Don't warn me again with this connector if you don't want this message to be displayed again, 
+   and then select **Continue**. 
+
+5. In the MongoDB Atlas SQL window that appears provide:
```
Comment (terakilobyte) [inline mongodb-atlas-sql-interface.md:47]:
```suggestion
5. In the MongoDB Atlas SQL window that appears, fill in the following values:
```

File: mongodb-atlas-sql-interface.md [hunk 4]
```diff
@@ -40,22 +40,25 @@ To connect using the Atlas SQL interface :
...
    - The **MongoDB URI** you retrieved previously. Make sure that it does not contain your username and password or it
```
Comment (terakilobyte) [inline mongodb-atlas-sql-interface.md:48]:
```suggestion
   - _Required_. The **MongoDB URI**. _Required_. 
   
      Use the MongoDB URI you obtained in step 1.  Make sure that it does not contain your username and password. URIs containing username and/or passwords will be rejected.
```

File: mongodb-atlas-sql-interface.md [hunk 5]
```diff
@@ -40,22 +40,25 @@ To connect using the Atlas SQL interface :
...
 
-2. In the MongoDB Atlas SQL window that appears provide:
+4. If this is the first time you're connecting to MongoDB Atlas SQL connector, a third-party notice will be displayed. 
+   Select Don't warn me again with this connector if you don't want this message to be displayed again, 
+   and then select **Continue**. 
+
+5. In the MongoDB Atlas SQL window that appears provide:
    - The **MongoDB URI** you retrieved previously. Make sure that it does not contain your username and password or it 
      will be rejected.
    - Your federated **Database** name.
```
Comment (terakilobyte) [inline mongodb-atlas-sql-interface.md:50]:
```suggestion
   -  Your federated **Database** name. _Required_
   
      Use the name of the federated database you obtained in step 1.
```

File: mongodb-atlas-sql-interface.md [hunk 6]
```diff
@@ -40,22 +40,25 @@ To connect using the Atlas SQL interface :
...
    - An optional **SQL statement**.
```
Comment (terakilobyte) [inline mongodb-atlas-sql-interface.md:51]:
```suggestion
   - A **SQL statement**.  _Optional_
```

**Review by terakilobyte — CHANGES_REQUESTED** [review]
A few suggestions to make it clearer for the user. In general, I would also add a link in step 1 to Atlas for user convenience.

**Comment by nbagnard** [comment]
> I would also add a link in step 1 to Atlas for user convenience

I didn't on purpose, because right now everything is geared toward Tableau and the JDBC driver, so if our timing for the UI is off a little bit then I am afraid that it will make things more confusing than helpful.

File: mongodb-atlas-sql-interface.md (hunk 4 above)
Comment (nbagnard) [reply mongodb-atlas-sql-interface.md]: I think you did not mean to have _required_ twice. I couldn't find how to edit your suggestion, but I made the change to mark it required 👍

File: mongodb-atlas-sql-interface.md [hunk 7]
```diff
@@ -34,22 +40,35 @@ To connect using the Atlas SQL interface :
 
 3. Select **Database** from the categories on the left, select **MongoDB Atlas SQL**, and then select **Connect**.
 
-2. In the MongoDB Atlas SQL window that appears provide:
-   - The **MongoDB URI** you retrieved previously. Make sure that it does not contain your username and password or it will be rejected.
-   - Your federated **Database** name.
```
Comment (alexiantonino) [inline mongodb-atlas-sql-interface.md]: I believe in Power BI you must enter in the database (even though it is also included in the string)

**Review by alexiantonino — APPROVED** [review]

**Comment by nbagnard** [comment]
evergreen merge

**Pull request merged**